core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -469,6 +469,16 @@ private static String toNormalizedIconUrl(String url) {
         icons.addIcon(new Icon("icon-up icon-md", "24x24/up.png", Icon.ICON_MEDIUM_STYLE));
```

Comment:
can't we remove these old icon's (at least the medium style)?

Reply:
I had left them there as I wasn't sure if they were needed for backwards compat - more than happy to remove them though, am I good to remove any .gifs of the same names too?

Reply:
Yes

Reply:
Awesome, those are all gone now 🚀 

Are the actual image files good to be deleted as well or would they need to stay for backwards compat?